sklearn/utils/extmath.py

```diff
@@ -508,6 +508,8 @@ def svd_flip(u, v, u_based_decision=True):
         u and v are the output of `linalg.svd` or
         :func:`~sklearn.utils.extmath.randomized_svd`, with matching inner
         dimensions so one can compute `np.dot(u * s, v)`.
+        The input v should really be called vt to be consistent with scipy's
```

Comment:
For those who specific `v` as keyword argument?

I am starting to see the argument for [pep 570](https://www.python.org/dev/peps/pep-0570/) in (python 3.8).

Reply:
Yes, I want to change it too but it's public :/

Definitely a good candidate for pep 570!